src/structures/voice/VoiceState.ts

```diff
@@ -0,0 +1,55 @@
+import { GatewayVoiceState, Snowflake } from 'discord-api-types';
+import { StageChannel } from '../channel/voice/StageChannel';
+import { VoiceChannel } from '../channel/voice/VoiceChannel';
+import { Guild } from '../guild/Guild';
+import { GuildMember } from '../guild/GuildMember';
+
+export class VoiceState {
+	public channelId?: Snowflake;
+	public userId!: Snowflake;
+	public member?: GuildMember;
+	public sessionId!: string;
+	public serverDeaf!: boolean;
+	public serverMute!: boolean;
+	public selfDeaf!: boolean;
+	public selfMute!: boolean;
+	public selfStream?: boolean;
+	public selfVideo!: boolean;
+	public suppress!: boolean;
+	public requestToSpeakTimestamp?: number;
+	public requestToSpeakAt?: Date;
```

Comment:
what is this?

Reply:
This is for StageChannel, users can request to speak.

Reply:
why it's on VoiceState if it's only for StageChannels?